spec/functional_tests/documents/filter/my_documents_filter_spec.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+admin = TestingAppServer::UserData.new
+api_admin = TestingAppServer::ApiHelper.new(admin.portal, admin.mail, admin.pwd)
+user = TestingAppServer::UserData.new(first_name: 'John', last_name: 'Smith',
+                                      mail: TestingAppServer::PrivateData.new.decrypt['user_mail'],
+                                      pwd: SecureRandom.hex(7), type: :user)
+api_admin.people.add_user(user) unless api_admin.people.user_with_email_exist?(user.mail)
+
+describe 'Documents filter My documents' do
+  before :all do
```

Comment:
I think this `before :all` can be safely extracted before `describe` on line 11 and test should work like before

But I don't think this can be done with `after :all` (